### DISCUSSION TITLE
symphony level set remake?

### DISCUSSION THREAD

**Discussion by clementtttttttt** [Neverball/neverball#365] [General]
decided to check in on the game and apparently there's a remake of my old levels?

**Comment by parasti** [comment]
So here's a little story.

A while ago I added a bunch of level sets to the in-game downloads.

I found this level set (SYMPHONY LEVEL REMAKE) on the old forums via Internet Archive.

Here's the link if you're interested: https://web.archive.org/web/20210811080911/http://neverforum.com/fmpbo/viewtopic.php?id=3441

Since the formatting in that link is mostly broken, here's also the text of the forum post:

---

* dtb
* OccasionallyBall
* Offline
* From: Vienna, Austria
* Registered: 2011-04-03
* Posts: 576

**Topic: Symphony Level Remake**

Hey, just in case someone missed this great set, now here's the accompanying story...

<strong>p2013125</strong> once posted a 30 levels set called SYMPHONY LEVEL!&nbsp; (yes, 30 levels - of which only 25 were displayed in-game due to the limit of that number). The levels also had no names but only were numbered. This set can be found here: <br><a href="http://neverforum.com/fmpbo/viewtopic.php?pid=31814#p31814">http://neverforum.com/fmpbo/viewtopic.p … 814#p31814</a><br>The set itself was enjoyable, rather medium to max. hard difficulty, some good ideas, some levels with a somehow unfinished look-and-feel (like early ht-never stuff)

Some time later, <strong>DASH MARKY </strong>posted&nbsp; a remix set called SYMPHONY LEVEL REMAKE (quite a large file) which can be found here:<br><a href="http://neverforum.com/fmpbo/viewtopic.php?pid=33656#p33656">http://neverforum.com/fmpbo/viewtopic.p … 656#p33656</a><br>When I first downloaded it back then and put it in my NB Folder nothing much happened. I probably considered it&nbsp; a complete miss and forgot about it pretty soon. Only recently I realized that both zip-files happened to contain the same editor in the text file ("set-sy") and this made the game display only the old set (SYMPHONY LEVEL!) in the levels menu overview. 

<strong>Now I repacked the SYMPHONY LEVELS REMAKE set and changed all of the "sy" related attributes to "syRemake", so both sets will show up in the overview. Attached here!</strong>

The levels of the Remake set are well polished, partly hard to beat and overall great fun to play through. Now I'm not really sure to whom the credits for this set go (p2013125 or DASH MARKY?) but I thank you for this - especially all of the bonus levels really made may day! <br>Favourite levels: 2, 7, 12, 17, 18, 20, II (which is hard but can be done) and IV (which is my favourite by far - and yes,&nbsp; it is possible to get all coins here wink)

Cheers

dtb

<strong id="attach35359">Post's attachments</strong>

<a href="http://neverforum.com/fmpbo/misc.php?action=pun_attachment&amp;item=1989&amp;download=1">SYMPHONY LEVEL REMAKE.zip</a>&nbsp;14.28 mb, 130 downloads since 2020-04-25&nbsp;</div>

**Reply by clementtttttttt** [reply]
i don't think i have ever simultaneously cringed and be proud of my 10 years old self so much before
